build/cwc/aiy.js

```diff
@@ -0,0 +1,135 @@
+  '../../third_party/aiyprojects-raspbian/src/examples');
+let outDir = path.join(__dirname,
+  './../../genfiles/core/resources/examples/aiy');
+
+let examples = [
+  'voice/voice_recorder.py',
+  'voice/assistant_grpc_demo.py',
+  'vision/joy/joy_detection_demo.py',
+  'vision/image_classification_camera.py',
+  'button_led.py',
+  'leds_example.py',
+
+  // TODO(carheden): These don't work with my voice kit, possibly
+  // due to hardware options
+  // 'gpiozero/bonnet_button.py',
+  // 'gpiozero/button_example.py',
+  // 'gpiozero/led_chaser.py',
+  // 'gpiozero/led_example.py',
+  // 'gpiozero/servo_example.py',
+  // 'gpiozero/simple_button_example.py',
+
+  // TODO(carheden): These require API keys, which we can't set up from CwC yet.
+  // 'buzzer/buzzer_demo.py',
+  // 'voice/assistant_library_demo.py',
+  // 'voice/assistant_library_with_button_demo.py',
+  // 'voice/assistant_library_with_local_commands_demo.py',
+  // 'voice/cloudspeech_demo.py',
+
+
+  // TODO(carheden): These examples require files in addition to the code.
+  // Figure out how to push additional files to the AIY or modify them to use
+  //  a single file.
+  // 'buzzer/buzzer_tracker_demo.py',
+  // 'vision/object_meter/object_meter.py',
+  // 'vision/object_meter/wordnet_grouping/mapping_data.py',
+  // 'vision/object_meter/wordnet_grouping/category_mapper.py',
+
+  // TODO(carheden): These expect command line agruments. Modify them to work
+  // from CwC
+  // 'vision/any_model_camera.py',
+  // 'vision/dish_classification.py',
+  // 'vision/dish_detection.py',
+  // 'vision/face_detection_camera.py',
+  // 'vision/face_detection.py',
+  // 'vision/face_detection_raspivid.py',
+  // 'vision/image_classification.py',
+  // 'vision/inaturalist_classification.py',
+  // 'vision/mobilenet_based_classifier.py',
+  // 'vision/object_detection.py',
+  // 'vision/video_capture/video_capture.py',
+  // 'vision/face_camera_trigger.py',
+];
+
+/*
+ * We have to encapsulate python code in for AIY in a .cwc file to
+ * force it to use the AIY CwC mode mode.
+ *
+ * TODO(fstanis/carheden/mbordihn): If user open an AIY python file from the
+ * menus, they won't be able to use AIY because cwc will load the wrong mode.
+ * Consider an 'activate AIY' button in the Python mode or something.
+ */
+let pyToAIYCwc = function(python) {
+  return {
+    'content': {
+      '__python__': {
+        'content': fs.readFileSync(python, 'utf8'),
+        'name': '__python__',
+        'size': fs.statSync(python).size,
+        'type': 'text/x-python',
+        'version': 1,
+      },
+    },
+    'files': {},
+    'flags': {},
+    'format': 'Coding with Chrome File Format 3',
+    'frameworks': {},
+    'history': '',
+    'metadata': {
+      '__default__': {
+        'author': 'AIY Projects Team',
+        'title': path.basename(python, '.py')
+          .replace(/_/g, ' ')
+          .replace(/ \w/g, (part) => part.toUpperCase())
+          .replace(/^\w/g, (part) => part.toUpperCase()),
```

Comment:
Could you separate the replaces to a new function like normalizeTitle or something like this.